blackbox/docs/sql/joins.txt

```diff
@@ -76,6 +76,65 @@ the other table::
     +----+------------+------------------+
     SELECT 4 rows in set (... sec)
 
+Left Joins
```

Comment:
Think we should go with the full name. `Left outer joins`. I can imagine people might be searching for "outer"